### ISSUE TITLE
Model diverged with loss = NaN

### ISSUE THREAD

**Issue by ttyang1018** [WeiTang114/MVCNN-TensorFlow#8]
Hi there,

I manually generated the 12 views of each airplane and bathtub png image, and train with them. But I encountered following model diverged problem. 

![image](https://user-images.githubusercontent.com/13607569/28679659-810e69d4-7326-11e7-906d-7064a93105e6.png)

Could someone help me out of this？

**Comment by yifanfeng97** [comment]
I have the same problem!

**Comment by foxfromworld** [comment]
Hi...
I wonder if this problem has been solved??
I have encountered the same problem...

**Comment by yifanfeng97** [comment]
I have solved my nan problem. My reason is that my dataset is converted from MVCNN(MATLAB code). The label in MATLAB is in range 1 and 40. So I reduced 1 on label converting it into range 0 and 39.

**Comment by foxfromworld** [comment]
Thank you so much for your quick response!
May I ask what the labels are? Type of objects?
I am sorry...I am very new to NN...
Thank you again!

**Comment by yifanfeng97** [comment]
yes, the label is a number to represent the name of objects in training.

**Comment by foxfromworld** [comment]
Thanks a million. Really appreciate your help! 
I will try to change the way I feed the data and see if it works!

**Comment by ghost** [comment]
> I have solved my nan problem. My reason is that my dataset is converted from MVCNN(MATLAB code). The label in MATLAB is in range 1 and 40. So I reduced 1 on label converting it into range 0 and 39.

Thanks a lot. I solved this by your comment. The reason is when I processed the data, I labeled the categories from 1 to 40. Changing them back to 0 to 39 fixes this problem.
